controller/oidc_auth/login.go

```diff
@@ -348,6 +349,12 @@ func (l *login) authenticate(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
```

Comment:
Does this need to go through the manager so it gets turned into a raft journal entry?

Reply:
The store here is the OIDC OP `store` which is defined in `storage.go` and does `return s.env.GetManagers().Identity.UpdateSdkEnvInfo(identity, envInfo, sdkInfo, changeCtx)`

The "store" naming here came from the interface defined by zitadel/oidc, and unfortunately, it conflicts with our intern "store" naming.

Reply:
ah, ok. Makes sense